scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -186,10 +187,7 @@ export function AuthoringDirective(
                                 $scope.stage = result;
                             });
 
-                        desks.fetchDeskById($scope.origItem.task.desk).then((desk) => {
-                            $scope.deskName = desk.name;
-                            $scope.deskType = desk.desk_type;
-                        });
+                        getDesk();
```

Comment:
Functions that are called `get{Something}` shouldn't **write** to variables outside the function. `setDesk` would be a better name.